### PR TITLE
test: Fix inline snapshot mismatch on certain machines

### PR DESCRIPTION
**Pull request by bpinto**
**What**:

Two snapshot tests were failing locally while passing on CI, this change fixes it so that tests are green on CI as well as locally.
 
The replaced library was appending extra double quotes at the begging and end of the string.

https://github.com/joaogranado/jest-serializer-ansi/blob/c8c5e8cbc0f804f9fe195dfd27625ea0b3d2a062/index.js#L13-L16

**How**:

Replacing jest-serializer-ansi library with jest-snapshot-serializer-ansi.

**Checklist**:

- [ ] Documentation added to the [docs site](https://github.com/testing-library/testing-library-docs) N/A
- [x] Tests
- [ ] TypeScript definitions updated N/A
- [x] Ready to be merged

### CODE REVIEW

File: src/__tests__/pretty-dom.js [hunk 1]
```diff
@@ -45,16 +45,34 @@ test('prettyDOM defaults to document.body', () => {
   </body>"
 `
   renderIntoDocument('<div>Hello World!</div>')
-  expect(prettyDOM()).toMatchInlineSnapshot(defaultInlineSnapshot)
-  expect(prettyDOM(null)).toMatchInlineSnapshot(defaultInlineSnapshot)
+  expect(prettyDOM()).toMatchInlineSnapshot(
```
Comment (bpinto) [inline src/__tests__/pretty-dom.js:48]: Hmmm... not sure what happened here, I need to investigate this change.

Comment (bpinto) [reply src/__tests__/pretty-dom.js]: Okay, fixed, it was a problem with automatically updating inline snapshots and then the previous expectation had the extra double quotes.

**Comment by eps1lon** [comment]
That's awesome! Will check tomorrow if this fixes it for me locally. And then we probably want to apply this fix in all testing-library repos

**Comment by bpinto** [comment]
Let me know how it goes! Regarding other testing-library repos, I could not find any other repository making use of ansi snapshot tests, I found this [PR](https://github.com/testing-library/jest-dom/pull/16) removing some snapshot tests on jest-dom repository, maybe this was experiencing a similar issue?

**Comment by timdeschryver** [comment]
If you need someone on a Windows environment to test this, feel free to ping me.

**Comment by bpinto** [comment]
@timdeschryver sorry if the comment wasn't for me, but if I may, I think running tests on Windows could do no harm and would, in fact, being more confidence to these changes. So please, if you could run on Windows, I think that would be great.

**Review by timdeschryver — APPROVED** [review]
Works on my machine 👍

**Review by eps1lon — APPROVED** [review]
Awesome!

**Pull request merged**